Example/Example/ExampleApp.swift

```diff
@@ -5,11 +5,24 @@ import SwiftUI
 
 @main
 struct ExampleApp: App {
+
+  // MARK: Lifecycle
+
+  init() {
+    // Register fonts from the Samples/Fonts directory
+    for fontAssetURL in Bundle.main.urls(forResourcesWithExtension: "ttf", subdirectory: "Samples/Fonts") ?? [] {
```

Comment:
Unrelated to this bugfix -- I added a test case for https://github.com/airbnb/lottie-ios/issues/1939, since we didn't have any example animations that used custom fonts.